### PR TITLE
[models] Change Resize kwargs to args for each zoo predictor

### PR DESCRIPTION
**Pull request by cmoscardi**
Closes https://github.com/mindee/doctr/issues/1764 -- more details in that issue. Hope this all helps!

### CODE REVIEW

**Review by felixdittrich92 — CHANGES_REQUESTED** [review]
Hi @cmoscardi :wave:,

Thanks for the PR :+1: 

I agree from user view it's cleaner but then we should keep it consistent over all `_predictor` instances and don't miss to update the docstrings :)

I created a branch as reference (feel free to copy from):
https://github.com/felixdittrich92/doctr/commit/c62b851cec2a9f626947b85536536e67fb5de050

File: doctr/models/detection/zoo.py [hunk 1]
```diff
@@ -94,10 +97,20 @@ def detection_predictor(
...
+        symmetric_pad: if True, pad the image symmetrically instead of padding at the bottom-right
+        batch_size: number of samples the model processes in parallel
         **kwargs: optional keyword arguments passed to the architecture
 
     Returns:
     -------
         Detection predictor
     """
-    return _predictor(arch, pretrained, assume_straight_pages, **kwargs)
+    return _predictor(arch,
```
Comment (felixdittrich92) [inline doctr/models/detection/zoo.py:110]: @cmoscardi 
arch=arch
pretrained=pretrained

**Comment by felixdittrich92** [comment]
@cmoscardi you can apply formatting with `make style` :)

**Comment by cmoscardi** [comment]
> @cmoscardi you can apply formatting with `make style` :)

Ah thank you! I am in a limited connectivity environment right now and will be able to get the dev dependencies installed in a few hours. I also wanted to run tests before pinging you, thanks for the quick review of this :)

**Comment by felixdittrich92** [comment]
> > @cmoscardi you can apply formatting with `make style` :)
> 
> Ah thank you! I am in a limited connectivity environment right now and will be able to get the dev dependencies installed in a few hours. I also wanted to run tests before pinging you, thanks for the quick review of this :)

No stress 😁 👍

**Comment by cmoscardi** [comment]
OK should be good now!

**Review by felixdittrich92 — APPROVED** [review]
LGTM thanks :+1:

**Pull request merged**